packages/contracts-bedrock/src/L2/SuperchainERC20Bridge.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+// Libraries
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+// Interfaces
+import { ISuperchainERC20 } from "src/L2/interfaces/ISuperchainERC20.sol";
+import { IL2ToL2CrossDomainMessenger } from "src/L2/interfaces/IL2ToL2CrossDomainMessenger.sol";
+
+/// @custom:proxied true
+/// @custom:predeploy 0x4200000000000000000000000000000000000028
+/// @title SuperchainERC20Bridge
+/// @notice The SuperchainERC20Bridge allows for the bridging of ERC20 tokens to make them fungible across the
+///         Superchain. It builds on top of the L2ToL2CrossDomainMessenger for both replay protection and domain
+///         binding.
+contract SuperchainERC20Bridge {
```

Comment:
Sorry for the naming nits, I know that I proposed this one, curious if you have any thoughts on the following issue: https://github.com/ethereum-optimism/specs/issues/399

Reply:
We agree on changing the name, we notice that `asset` is often used

so to be more consistent, do u prefer `SuperchainTokenBridge` or `SuperchainAssetBridge`?

Reply:
I have a slight preference for `SuperchainTokenBridge` unless you think that there are non token assets?